Directory.Build.props

```diff
@@ -10,7 +10,7 @@
   <PropertyGroup>
     <AssemblyVersion>2.0.0.0</AssemblyVersion>
     <FileVersion>2.0.0.0</FileVersion>
-    <Version>2.121.101</Version>
+    <Version>2.121.102</Version>
```

Comment:
Don't forget to merge from main in Avalonia branch

Reply:
This is for Avalonia branch only.